lib/grammers-client/src/client/messages.rs

```diff
@@ -546,6 +546,119 @@ impl Client {
         })
     }
 
+    /// Sends a album to the desired chat.
+    ///
+    /// This method can also be used to send a bunch of media such as photos, videos, documents, polls, etc.
+    ///
+    /// If you want to send a local file as media, you will need to use
+    /// [`Client::upload_file`] first.
+    ///
+    /// Refer to [`InputMedia`] to learn more formatting options, such as using markdown.
+    ///
+    /// See also: [`Message::respond_album`], [`Message::reply_album`].
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # async fn f(chat: grammers_client::types::Chat, client: grammers_client::Client) -> Result<(), Box<dyn std::error::Error>> {
+    /// use grammers_client::InputMedia;
+    ///
+    /// client.send_album(&chat, vec![InputMedia::caption("A album").photo_url("https://www.pixelstalk.net/wp-content/uploads/2016/03/Free-cute-animal-wallpaper-HD.jpg")]).await?;
```

Comment:
Prefer using a URL like `https://example.com/cat.jpg`.

Any other domain could go down, be hijacked, or a lot of different things. `example.com` is defined for explicit use in examples.

Reply:
sure